package-lock.json

```diff
@@ -1,12 +1,11 @@
 {
     "name": "zod-http-schemas",
-    "version": "2.0.3",
+    "version": "2.0.1",
```

Comment:
The latest version is `2.0.3`, this should either match the package.json file or we should update all versions to 2.0.4 for the next release we'll do.

Reply:
right thank you